build/build.sh

```diff
@@ -18,3 +18,6 @@ rm dist/cli.js
 
 # types
 cp types/types.d.ts dist/types.d.ts
+
+# create browser build for docs
+npx terser dist/browser.js --compress --mangle --ecma 2015 --output civet.dev/public/__civet.js
```

Comment:
If we're going to use `terser` we should add it as a dev dependency instead of using `npx` to lock the version.

Reply:
I should have read further, it looks like it was already added so we should be able to just call it directly as `terser dist/browser.js --compress --mangle --ecma 2015 --output civet.dev/public/__civet.js`

Reply:
You are right. Fixed